fdbclient/BlobCipher.cpp

```diff
@@ -1043,9 +1042,10 @@ Reference<EncryptBuf> EncryptBlobCipherAes265Ctr::encrypt(const uint8_t* plainte
 	if (authTokenMode != ENCRYPT_HEADER_AUTH_TOKEN_MODE_NONE) {
 		header->cipherHeaderDetails = headerCipherKeyOpt.get()->details();
 	} else {
-		header->cipherHeaderDetails.encryptDomainId = INVALID_ENCRYPT_DOMAIN_ID;
-		header->cipherHeaderDetails.baseCipherId = INVALID_ENCRYPT_CIPHER_KEY_ID;
-		header->cipherHeaderDetails.salt = INVALID_ENCRYPT_RANDOM_SALT;
+		header->cipherHeaderDetails = BlobCipherDetails();
+		ASSERT_EQ(INVALID_ENCRYPT_DOMAIN_ID, header->cipherHeaderDetails.encryptDomainId);
```

Comment:
nit:  default constructor sets values are INVALID; we can remove the assert statements

Reply:
just to make sure there's no surprises. and this is for the old encryption header format which will soon gone.

Reply:
may be replace it with `assert !header->cipherHeaderDetails.isValid()`